### PR TITLE
Adding base64 encoding

### LINKED ISSUE #1295: Running a local html sample throws CORS errors regarding the new fonts/icons
### Description  
Running an html sample page as a local file and not through IIS throws CORS(Cross-Origin Resource Sharing) errors regarding the new fonts/icons.

### Steps to reproduce  

1. Open the attached sample
2. Observe the errors thrown

### Result  
There are errors thrown.
Due to this reason no icons are visible.

### Expected result  
There should be no errors. Icons/fonts should be able to be loaded through Cross-Origin Resource Sharing mechanism.

**Note:** This is a general issue that probably does not affect only the grids.

**How to fix:**
In order to be fixed the @font-face inside the **icon-style.css** files should load files through **base64 encoding** as well.

### Attachments  
[igGrid-Overview.zip](https://github.com/IgniteUI/ignite-ui/files/1363055/igGrid-Overview.zip)

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #1295

### CODE REVIEW

File: src/css/themes/bootstrap3/default/icons-styles.css [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
...cuATc+ARceAQI2HHAzNCEbHG80MyLfF1wqKhwXFlwqKxvaFFMmJxkVFFMmJxn+2xJMIiMXExJLIyMXsxFEIB8VERFEHyAUECcPPh0cEw8QPR0cEw8BAQ44GRoRDg04GhoRDgK/GWYuLx8ZGWYuLx8DtTQiHBtwNDMiHBtw/esqHBcWXCorGxYXXKknGRUUUycmGRQVU0EjFxMSTCIjFxMSTAEYIBQQEUQgHxUREUQfAT8cEw8PPh0cExAPPR3KGhEODjgaGRENDjga/oguHxkZZS8uHxkZZQAAAAEAJwAnBAAEAABAAAABIg4CBw4BHgEXHgI2Nz4DNTA0NTQmIyIGFRwBMRQOAgcOAS4BJy4CNjc+AzMwMjMyNjU0JiMqATECAE6Re2EeHg4dRzc3hpKXSEh0Uy0gFxcgI0FcODl2c2krLDcXCxcYTGFyPQEBFiAgFgEBBAAtU3RISJeShjc3Rx0OHh5he5FOAQEWICAWAQE9cmFMGBcLFzcsK2lzdjk4XEEjIBcXIAAAAgAAABEEAAPvADoAcgAAAS4BJy4BJy4BJy4BBw4BBw4BBw4BBw4BFx4BFx4BFx4BFx4BNz4BNz4BNz4BNz4BNzoBMzI2NTwBNTEHDgEHDgEHDgEnLgEnLgEnLgEnLgE3PgE3PgE3PgE3PgEXHgEXHgEXHgEXHgEHMRwBFRQWFw4BBwQAARYVFTslJFcvL2QyM2IuLlEjIjUSEhEBARUUEzcjI1AtLF4vL1wqK0wgIDIRCg0EAQIBGydpEjQgIUspKVgrLFYoJ0ceHi4PDw8BARMREDEeHkYmJlEpKE8lJUEcGysODg0BIhkFEAsB7zRmLy9VIyQ3EhMRAQEVFBQ6IyRULS5hMTBfLSxPISE0EREQAQEUExM1IiFOKxo2GycbAQMBrylKHx4wEBAPAQETERIyHx9JKCdUKitSJiZFHB0sDw8NAQERERAvHB1EJCVNKAEDARolAxs0GQAADAAAAAAEAAQAABAAMABQAHAAkAChALIA0gDyARIBMgFDAAABIgYdARQWOwEyNj0BNCYrAQciBg8BDgEVFBYfAR4BMzI2PwE+ATU0Ji8BLgEjMCIxITgBMSIGDwEOARUUFh8BHgEzMjY/AT4BNTQmLwEuASMFOAExIgYPAQ4BFRQWHwEeATMyNj8BPgE1NCYvAS4BIyEiBg8BDgEVFB
```
Comment (skrustev) [inline src/css/themes/bootstrap3/default/icons-styles.css]: For ttf should be "data:application/octet-stream;" instead of "data:application/font-ttf;"

Comment (skrustev) [inline src/css/themes/bootstrap3/default/icons-styles.css]: For woff should be "application/font-woff"

Comment (skrustev) [inline src/css/themes/bootstrap3/default/icons-styles.css]: For svg should be "image/svg+xml;charset..."

Comment (skrustev) [inline src/css/themes/bootstrap3/default/icons-styles.css]: It is still throwing an error for the jquery-ui.ttf.
I would recommend creating a duplicate @font-face style with just the base64 encoding urls and leave the first @font-face like before.

**Comment by kdinev** [comment]
The base64 encoding makes the styles too large. The font files is what font would be delivered through.

**Pull request closed without merging**